working/augmentation-libraries/feature-specification.md

```diff
@@ -269,9 +269,8 @@ it if they know what they’re doing.
 
 ### Type inheritance
 
-When a type is omitted in an augmenting declaration, if the first declaration
-that introduced the member has a type specified in the ommitted types position,
-then that type is inherited.
+When a type is omitted in an augmenting declaration, it inherits the final type
```

Comment:
"final type" is not a defined concept.
Also, a type can be omitted which is not the type of the entire augmented declaration.

I'd make the type inheritance part of augmentation application. The augmenting declaration does not specify a type for something. Applying an agumenting declaration with types omitted will preserve the existing type of that thing from the augmented declaration.
(And an augmenting declaration which *does* specify a type for something must specify the same type as the augmented declaration has for that thing, which means the application still preserves the type of the augmented declaration. Types can be written in augmentations, but they cannot change anything, they have no effect other than documentation, and being validated for being the correct type.)

Consider:
> When a type is omitted in an augmenting declaration, be it a return type, variable type, parameter type,
> or type parameter bound type, the result of applying the augmentation has the same type at that position
> as the augmented declaration. _An augmenting declaration "inherits" omitted types from the augmented
> declaration._